.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
```

Comment:
Should we name the workflow file and workflow itself like `fast_testing` or `testing`?

Reply:
`test` is more than enough.